integration-tests/bats/sql-client.bats

```diff
@@ -37,7 +37,7 @@ setup() {
 }
 
 teardown() {
-    stop_sql_server
+    stop_sql_server 1 && sleep 0.5
```

Comment:
Nit: Can we `wait` the pid we killed, or is this send metrics nonsense? A comment about why?

Reply:
I think this was `send-metrics` nonsense, and is now fixed